.ci/set_kata_config.sh

```diff
@@ -35,18 +46,30 @@ Remove it if you want to use the stateless options.
 EOT
 }
 
-case ${request} in
-	true)
-		;;
-	false)
-		;;
-	*)
-		usage
-		exit 1
-		;;
-esac
-
-current_value=$(kata-runtime kata-env --json | jq ".Runtime.SandboxCgroupOnly")
+if [[ -z "${option}" || ! ${!valid_requests[*]} =~ ${option} ]]
+then
+	# unknown option
+	usage
+	exit 1
+fi
+
+if [[ -z "${request}" || ! ${valid_requests["${option}"]} =~ ${request} ]]
+then
+	# unknown request
+	usage
+	exit 1
+fi
+
+# quote non-boolean request
+if [[ "${request}" != "true" && "${request}" != "false" ]]
```

Comment:
You could simplify this to something like:

```bash
[[ "$request" =~ ^(true|false)$ ]] || request="\"${request}\""
```

Reply:
Updated.